data/mods/Mundane_Zombies/modinfo.json

```diff
@@ -66,7 +66,9 @@
       "mon_zombie_gasbag_fungus",
       "mon_zombie_smoker_fungus",
       "mon_skeleton_hulk_fungus",
-      "mon_skeleton_electric"
+      "mon_skeleton_electric",
+      "mon_zombie_kevlar_1",
+      "mon_zombie_kevlar_2"
```

Comment:
mon_zombie_soldier_blackops_1 has the `"CLASSIC"` flag so I think it is supposed to be somewhat normal zombie. Other than being stronger soldier it has good (too good?) night vision. mon_zombie_soldier_blackops_2 is just stronger and has better sense of smell.

I don't see anything in the monster that would connect to its description "This soldier zombie's skin radiates a sort of shadowy smoke that obscures its outline, making its camouflage even harder to see against the background.  In the darkness it would be nigh invisible."